### ISSUE TITLE
File info and preview in second pane.

### ISSUE THREAD

**Issue by MeanEYE** [MeanEYE/Sunflower#61]
In order to manage files (view, change names, delete, move, etc) such as scans of paperwork and photos I currently open each file separately to view, and right click to find its (meta-) information.The results usually open in new windows outside my file manager. All this opening and closing of windows for each file make it rather cumbersome to manage large numbers of files efficiently.

Would it be possible to enhance Sunflower-fm with a function that allows the right pane to provide a (readable/zoomable) preview and file info (maybe in separate tabs) of a file selected in the left pane?

Currently I do not know of any linux filemanager that can do this, I imagine this to be a great function for users that are heavy file manager users.

**Comment by amitbha** [comment]
Good idea！similar to gnome-sushi preview, used in nautilus
http://ubuntuguide.net/enable-nautilus-preview-ubuntu-1310

**Comment by darkelement** [comment]
This is great idea. Couple of years ago I tried to smuggle it in, but didn't find much attention.

Two years ago I started working on making all windows dockable in panes, but stopped it because of plans to migrate Sunflower to gtk3. When gtk3 will be ready I can revive this work.

**Comment by dufferzafar** [comment]
Yeah, [ranger](https://github.com/hut/ranger/) does this too. It's written in Python as well, so we could definitely see how they achieve it. 

It uses a file [scope.sh](https://github.com/dufferzafar/dotfiles/blob/master/ranger/scope.sh) which checks what the mimetype of the file is and based on it, 'decides' what to show in the second pane.

So for example if the file is pdf, it'll display a `pdftotext` version of it.

**Comment by MeanEYE** [comment]
We don't need to check mime type through script. We have GIO for that. As for individual file type support, I would prefer that done on per-type basis like view is built to support now. Although viewer is not all that polished.

**Comment by strangeattr** [comment]
> Currently I do not know of any linux filemanager that can do this

Double Commander can. Double Commander seems to be by-design a direct clone (although fairly far from perfect actually) of Total Commander and this feature is already implemented in both of them. The default hot key is Ctrl+Q but it doesn't seem to work in DC, I've re-mapped it to Ctrl+E and it started working.
